instrumentation/servlet-user-5.0/build.gradle

```diff
@@ -0,0 +1,19 @@
+
+dependencies {
+    implementation(project(":agent-bridge"))
+    implementation('jakarta.servlet:jakarta.servlet-api:5.0.0')
+}
+
+jar {
+    manifest { attributes 'Implementation-Title': 'com.newrelic.instrumentation.servlet-user-5.0', 'Enabled': 'false' }
+}
+
+verifyInstrumentation {
+    passesOnly 'jakarta.servlet:jakarta.servlet-api:[5.0.0,)'
```

Comment:
I take it this instrumentation applies to servlet 6.0?

Reply:
VerifyInstrumentation did pass for 6.0.